### PR TITLE
Allow switching to default

### PR DESCRIPTION
**Pull request by kbond**

### CODE REVIEW

File: src/Routing/Router.php [hunk 1]
```diff
@@ -44,6 +44,12 @@ public function generate($name, $parameters = [], $referenceType = self::ABSOLUT
      */
     private function generateI18n($name, $parameters = [], $referenceType = self::ABSOLUTE_PATH)
     {
+        if (array_key_exists('_locale', $parameters) && $parameters['_locale'] === $this->defaultLocale) {
```
Comment (umpirsky) [inline src/Routing/Router.php]: @kbond Maybe we should move to `generate()`? WDYT?

**Review by umpirsky — APPROVED** [review]
Yeah, we need this. thanks!

File: src/Routing/Router.php (hunk 1 above)
Comment (kbond) [reply src/Routing/Router.php]: Sure, makes sense as this is not generating a i18n url.

Comment (kbond) [reply src/Routing/Router.php]: although, the same could be said of the condition below...

Comment (umpirsky) [reply src/Routing/Router.php]: In deed, I'll polish it afterwards, thanks!

**Pull request merged**